PeerioTechnologies/peerio-mobile
# feat: open share dialog when saving account key [ch12860]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/12860/mobile-backup-account-key-default-action-should-open-preview-and-trigger-share-dialog
 
![image](https://user-images.githubusercontent.com/750359/44803801-7ff06700-ab8d-11e8-8680-2b3ba86201b7.png)

#### Testing instructions  
when saving account key, preview and share action should appear, consequentially
file shared through "save key" should be readable.
tapping on the viewer should close share sheet, but it should be possible to share again

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
